cmd/scenario.go

```diff
@@ -25,7 +25,10 @@ func newScenarioListCommand(logger *zap.SugaredLogger) *cobra.Command {
 
 			fmt.Println("Available scenarios:")
 			for _, s := range manifest.Scenarios {
-				fmt.Println("ID: " + s.Id + ", Name: " + s.DisplayName)
+				fmt.Println("")
```

Comment:
Sam, I believe this change will print a hard-to-read list as it is jumping a line per attribute.

Reply:
It looks like this:
```
Available scenarios:

Name: Container Ambush
Description: Your CI build agent is compromised, where can you get from here?
ID: container-ambush

Name: Container Defeat in Detail
Description: A minor DaemonSet misconfiguration has led to cluster compromise, how?
ID: container-defeat-in-detail

Name: Container Phalanx Formation
Description: Use kubectl to investigate a logging bug that may be leaking secrets.
ID: container-phalanx-formation
...
```
I found that moving to multiple lines per scenario without a clear delimiter (space or colour) is quite hard to read personally. I think this is a nice compromise.